test/components/deprecated-tag.test.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ComponentDefinition } from '../../src/components/interfaces';
+import { buildProject } from './test-helpers';
+
+let component: ComponentDefinition;
+beforeAll(() => {
+  const result = buildProject('deprecated-tag');
+  expect(result).toHaveLength(1);
+
+  component = result[0];
+  console.log(component);
+});
+
+test('should have correct region definitions', () => {
+  expect(component.properties).toEqual([
+    {
+      defaultValue: undefined,
+      deprecatedTag: 'Custom CSS is not supported.',
+      description: 'Adds the specified classes to the root element of the component.',
+      inlineType: undefined,
+      name: 'className',
+      optional: true,
+      type: 'string',
+      visualRefreshTag: undefined,
+    },
+    {
+      defaultValue: undefined,
+      deprecatedTag: undefined,
+      description: 'Header',
+      inlineType: undefined,
+      name: 'header',
+      optional: true,
+      type: 'string',
+      visualRefreshTag: undefined,
+    },
+  ]);
+  expect(component.regions).toEqual([
```

Comment:
Can we test that regions can be deprecated as well?

Reply:
Updated! 👍